src/WorkItemMigrator/WorkItemImport/Agent.cs

```diff
@@ -903,12 +904,12 @@ public bool ImportRevision(WiRevision rev, WorkItem wi)
             }
         }
 
-        private void EnsureClasificationFields(Migration.WIContract.WiRevision rev)
+        private void EnsureClasificationFields(WiRevision rev)
         {
-            if (!rev.Fields.Any(f => f.ReferenceName == WiFieldReference.AreaPath))
+            if (!rev.Fields.HasAnyByRefName(WiFieldReference.AreaPath))
                 rev.Fields.Add(new WiField() { ReferenceName = WiFieldReference.AreaPath, Value = "" });
 
-            if (!rev.Fields.Any(f => f.ReferenceName == WiFieldReference.IterationPath))
+            if (!rev.Fields.HasAnyByRefName(WiFieldReference.AreaPath))
```

Comment:
This path should be for IterationPath, yes?

Reply:
Good catch, it should be iteration path.